app/controllers/statistics_controller.rb

```diff
@@ -0,0 +1,17 @@
+class StatisticsController < ApplicationController
+  def count_metric
+    count_metric_snapshot = MetricSnapshot.where(code: params[:metric_code]).count
+  	total_configuration = MetricConfiguration.all.count
+
+    if(total_configuration == 0)
+      metric_percentage = 0
+    else
+      metric_percentage = (count_metric_snapshot.to_f / total_configuration.to_f) * 100
```

Comment:
Don't you think that by removing the if/else  and keeping just this line would not be easier to understand?

The computational cost in practice of a multiplication and a division are not perceptible for the end user.